lib/tezos/lib/assets/copy-data-to-s3.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+set +e
+
+source /etc/environment
+aws s3 sync ~/.tezos-node/ s3://$S3_SYNC_BUCKET/
```

Comment:
You can use a faster `s5cmd` tool than the original aws s3 sync. We installed it earlier in `node.sh` on lines 68-72.

Reply:
✅

Reply:
✅